test-runner-frontend/src/Views/Report.js

```diff
@@ -286,7 +311,7 @@ export default function Report() {
                       <Box key={i} sx={{ width: '100%', display: 'flex', flexDirection: 'row' }}>
                         <Box key={i} sx={{ marginBottom: '3.6%', marginRight: '10%' }}>
                           <Typography variant="h5" id={`${currentChart[0]}`} sx={{ display: 'block' }}>
-                            <b>{currentChart[0]}</b>
+                            <b>{currentChart[1][0].benchmarkClass}</b>
```

Comment:
how is this data setup? why are we using the second item in `currentChart` and then the first item?

Reply:
`currentChart` is an array where the first element is the class name plus the mode abbreviation (i.e. AddError-thrpt) and the second element is an array containing all benchmarks for that class. We use the first element (first benchmark) because we want only the name of the class without the abbreviation (i.e. AddError), but we could use any other element (i.e. `currentChart[1][1]`).